jdi-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/RadioSteps.java

```diff
@@ -27,8 +26,8 @@ public void selectRadioIndex(int index, String name) {
 
     //#region Then
     @Then("^the \"([^\"]*)\" consists of next values:$")
-    public void theConsistOfNextValues(String name, List<String> values) {
-        assertEquals(radioButtons(name).values(), values);
+    public void theConsistOfNextValues(String name, String[] values) {
+        radioButtons(name).has().values(values);
```

Comment:
it's our changes? if yes - try to revert

Reply:
Changes from master